release-content/migration-guides/rename_spawn_gltf_material_name.md

```diff
@@ -7,3 +7,5 @@ pull_requests: [19287]
 When loading a Gltf scene in Bevy, each mesh primitive will generate an entity and store a `GltfMaterialName` component and `Name` component.
 
 The `Name` components were previously stored as mesh name plus primitive index - for example, `MeshName.0` and `MeshName.1`. To make it easier to view these entities in Inspector-style tools, they are now stored as mesh name plus material name - for example, `MeshName.Material1Name` and `MeshName.Material2Name`.
+
+Added a `GltfMeshName` component to ensure that behaviors that rely on Mesh's Name still work properly. Implemented core::ops::Deref for `GltfMeshName` and `GltfMaterialName` to ensure that they are used in the same way as `Name` (using equality comparison)
```

Comment:
```suggestion
If you were relying on the previous value of the `Name` component on meshes, use the new `GltfMeshName` component instead.
```

Just a suggestion -- The current incarnation of this migration guide entry has a lot of information but is not actually telling users how to fix their code.